__tests__/app.test.js

```diff
@@ -35,3 +35,24 @@ describe('GET /api/topics', () => {
           });
       });
 })
+
+describe('GET /api', () => {
+    test('status:200, responds with a JSON object containing an array of /api endpoint objects, each of which contains a property that describes what the endpoint serves', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+
+        const { endpoints } = body;
+
+        expect(Object.keys(endpoints).length).toBeGreaterThanOrEqual(3)
+
+        for (const endpoint in endpoints) {
+            const properties = endpoints[endpoint];
+            expect(properties).toHaveProperty("description", expect.any(String));
```

Comment:
You could be more specific here and just test that the response is the same as the file contents